pkg/controller/awsfederatedaccountaccess/awsfederatedaccountaccess_controller.go

```diff
@@ -147,8 +147,18 @@ func (r *ReconcileAWSFederatedAccountAccess) Reconcile(request reconcile.Request
 		return reconcile.Result{}, err
 	}
 
+	//TODO() will be refactored in Rafs pr to use secret
+	// create policy arn
+	accountCR := &awsv1alpha1.Account{}
```

Comment:
What are these changes for? I don't see anything in the PR description about changing the account access controller